app/indices/account_index.rb

```diff
@@ -27,4 +27,5 @@
     has 'CRC32(accounts.state)', as: :state, type: :integer
   end
 
+  where 'COALESCE(accounts.master, FALSE) = FALSE'
```

Comment:
That won't work on Oracle. There is no Boolean type. It is up to you, so better use 1 and 0

Reply:
yep, that could be rewritten.